### DISCUSSION TITLE
Code clarity

### DISCUSSION THREAD

**Discussion by Shubham0324** [ActivityWatch/activitywatch#583] [General]
Here: https://github.com/ActivityWatch/activitywatch/blob/master/scripts/logcrawler.py
Code from if statement at Line 19 is always false, so why we need it? 
```
        if False:
            print("Files: ")
            for filename in files:
                print(" - " + filename)
```

Sorry, I am just a beginner hence asking this kind of doubts.

**Comment by ErikBjare** [comment]
It was probably just used in debugging, you're correct that we don't need it. 

Feel free to submit a PR that removes it :)
